### ISSUE TITLE
Custom Person Card

### ISSUE THREAD

**Issue by hAMMERlized** [neilimixamo/Home-Assistant-Quick-Look-Mobile#13]
Hi,

a person card with some detailed info would be nice (travel time, mobile phone battery status, which zone you are etc.). 

A good example can be found here:

https://github.com/UI-Lovelace-Minimalist/UI/tree/main/custom_cards/custom_card_person_info

**Comment by kesm** [comment]
I agree it could be nice to have more info on the card and the style is matching QLM. 
Another suggestion is to have the possibility to have a card different from person. On my previous dashboard I have a custom person card with zone, battery status, travel time and possibility to find my phone by making it ring louder.

I would love to be able to do this again
